templates/bbb-web.properties

```
@@ -37,6 +37,7 @@ breakoutRoomsPrivateChatEnabled={{ bbb_breakout_rooms_privatechat_enabled | tern
 userInactivityInspectTimerInMinutes={{ bbb_user_inactivity_inspect_timer }}
 userInactivityThresholdInMinutes={{ bbb_user_inactivity_threshold }}
 userActivitySignResponseDelayInMinutes={{ bbb_user_activity_sign_response_delay }}
+learningDashboardEnabled={{ bbb_learningDashboardEnabled }}
```

Variable has wrong casing here after renaming. (consider my other comment first)
```suggestion
learningDashboardEnabled={{ bbb_learningdashboardenabled }}
```

---

good catch.
Both done.

---

I would only set this option if you are on BBB 2.4.